azure-pipelines.yml

```diff
@@ -77,7 +77,7 @@ stages:
             vmImage: 'ubuntu-20.04'
           e2e_pool_definition:
             vmImage: 'ubuntu-20.04'
-          environment: PROFILE="-Dhadoop.version=2.8.5 -Dinclude_hadoop_aws -Dscala-2.12 -Darchunit.freeze.default.allowStoreUpdate=false"
+          environment: PROFILE="-Dhadoop.version=2.8.5 -Dinclude_hadoop_aws -Dscala-2.12 -Darchunit.freeze.store.default.allowStoreUpdate=false"
```

Comment:
Since we want this to be applied for all tests you could add this to `test_controller.sh#MVN_TEST_OPTIONS`.

Reply:
Ah, thanks. I'll move it there and re-add the violation entry to let CI confirm again that it does indeed still work.